shell/browser/ui/views/win_caption_button.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2021 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#ifndef SHELL_BROWSER_UI_VIEWS_WIN_CAPTION_BUTTON_H_
+#define SHELL_BROWSER_UI_VIEWS_WIN_CAPTION_BUTTON_H_
+
+#include "chrome/browser/ui/view_ids.h"
+#include "ui/base/metadata/metadata_header_macros.h"
+#include "ui/gfx/canvas.h"
+#include "ui/views/controls/button/button.h"
+
+namespace electron {
+
+class WinFrameView;
+
+class WinCaptionButton : public views::Button {
```

Comment:
Just curious if that could be used instead? https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/views/frame/windows_10_caption_button.cc

Reply:
Unfortunately the parameter `GlassBrowserFrameView* frame_view` makes this not practical to use as `GlassBrowserFrameView` would need a number of modifications in order to work with Electron.

Reply:
Yeah makes sense